Services/Notifications/classes/Setup/ilNotificationUpdateSteps.php

```diff
@@ -168,4 +168,16 @@ public function step_9(): void
         );
 
     }
+
+    public function step_10(): void
+    {
+        if (!$this->db->tableColumnExists('notification_osd', 'identification')) {
+            $this->db->addTableColumn('notification_osd', 'identification', [
+                'type' => 'text',
+                'length' => 255,
+                'notnull' => true,
+                'default' => ''
```

Comment:
Therefore I suggest to omit the default.

Reply:
See above.
That default is the last stand to ensure the the data consistency but nothing more (its like the trash scope).
I would not want to encourage someone to really using it by giving it a semantic default.